src/persist-client/Cargo.toml

```diff
@@ -39,8 +39,11 @@ futures-util = "0.3"
 mz-ore = { path = "../ore" }
 mz-persist-types = { path = "../persist-types" }
 mz-persist = { path = "../persist" }
+mz-repr = { path = "../repr" }
 proptest = { git = "https://github.com/MaterializeInc/proptest.git", default-features = false, features = ["std"] }
 proptest-derive = { git = "https://github.com/MaterializeInc/proptest.git" }
+prost = "0.10.1"
+rand = { version = "0.8.5", features = ["small_rng"] }
```

Comment:
not sure what this is for, but feels like at the very least it should be a dev-dependency